apps/frontend/src/utils/helpers.ts

```diff
@@ -112,6 +112,18 @@ export const validateEmail = (email: string) => {
   return re.test(email);
 };
 
+export const validateURL = (link: string) => {
+  const httpUrl =
```

Comment:
I think you have the naming the other way around - this regex doesn't validate URLs starting with `http(s)`.

Also, I wouldn't name this regex `wwwUrl` as it's a bit misleading, I'd choose something like `protocolRelativeUrl`.